frontends/pytorch/python/torch_mlir_utils/codegen/torch_signature_ods_gen.py

```diff
@@ -75,6 +76,55 @@ def generate_ops(g: "OpGenerator"):
     g.ordinary_unary_op(f"aten::{uname}(Tensor)",
                         f"{snakecase_to_camelcase(uname)}Op", uname)
 
+  # Convolution ops. Note that these are special in PyTorch and the importer,
+  # and we model them after the signatures of teh convolution_overrideable
+  # ops (generic for non-CPU/GPU backends) but set the names according to
+  # how they come in.
+  g.print_banner("NN ops")
+  g.ordinary_immutable_op(
+      "aten::convolution_overrideable(Tensor,Tensor,Tensor?,int[],int[],int[],bool,int[],int)",
+      "ConvolutionOp",
+      "convolution",
+      alias_kernel_names=["aten::convolution"])
+  g.ordinary_immutable_op(
+      "aten::convolution_backward_overrideable(Tensor,Tensor,Tensor,int[],int[],int[],bool,int[],int,bool[])",
+      "ConvolutionBackwardOp",
+      "convolution_backward",
+      alias_kernel_names=["aten::convolution_backward"],
+      # These do return as None but are not coded optional in the registry :(
+      override_return_types=["Tensor?", "Tensor?", "Tensor?"])
+
+  g.ordinary_immutable_op("aten::_log_softmax(Tensor,int,bool)",
+                          "LogSoftmaxImplOp", "log_softmax.impl")
+  g.ordinary_immutable_op(
+      "aten::_log_softmax_backward_data(Tensor,Tensor,int,Tensor)",
+      "LogSoftmaxBackwardDataImplOp", "log_softmax_backward_data.impl")
```

Comment:
The distinction I was trying to make was that these that are prefixed with "_" show up in ACAP traces because they are executed on the device but typically will be shadowed by a higher level op in other forms. I've just left this to resolve another day once we have a conflict and removed the suffix.